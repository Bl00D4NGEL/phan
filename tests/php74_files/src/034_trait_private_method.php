<?php
trait T34 { private abstract function private(); }
