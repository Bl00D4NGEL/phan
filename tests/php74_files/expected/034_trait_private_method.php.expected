%s:2 PhanCompatibleAbstractPrivateMethodInTrait Trait \T34 declares abstract private function private which is only allowed in 8.0+
